Gemfile

```diff
@@ -58,18 +55,18 @@ gem 'sprockets-rails', require: 'sprockets/railtie'
 gem 'statsd-instrument'
 gem 'twitter-text'
 gem 'tzinfo-data'
+gem 'webpacker', github: 'rails/webpacker'
```

Comment:
Using the `github:` style here produces a security warning on bundle install -- using `https://....` will fix it.

But also, I think this is on rubygems now: https://rubygems.org/gems/webpacker/versions/1.2